src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/MediaSystem.cs

```diff
@@ -113,28 +112,13 @@ internal static bool ConnectChannels(MediaContext mc)
         private static void ReadAnimationSmoothingSetting()
         {
 #if PRERELEASE
```

Comment:
Is this set anywhere?  I don't see it in the csproj, props, or targets.  This is probably dead code.  We don't have to get rid of it just now if it feels better to keep it, just an observation.